src/tsolvers/lrasolver/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@ target_sources(tsolvers
     PRIVATE "${CMAKE_CURRENT_LIST_DIR}/Delta.h"
     PRIVATE "${CMAKE_CURRENT_LIST_DIR}/LABounds.h"
     PRIVATE "${CMAKE_CURRENT_LIST_DIR}/LARefs.h"
+    PRIVATE "${CMAKE_CURRENT_LIST_DIR}/LAVarMapper.h"
```

Comment:
Should we move these files regarding LARefs, LAVarMapper, LABounds, etc. to the LASolver folder now?

Reply:
I think that would be good.  I was hesitant to do this straight up because of the parallel development on LIA solver, but perhaps this is not an issue in reality.

Reply:
Done.